lib/taskana-core/src/main/java/pro/taskana/impl/TaskServiceImpl.java

```diff
@@ -612,6 +613,33 @@ public void forceDeleteTask(String taskId)
     }
   }
 
+  @Override
+  public Task setOwner(String taskId, String ownerId)
+      throws TaskNotFoundException, InvalidStateException, NotAuthorizedException {
+
+    LOGGER.debug("entry to setOwner(taskId = {}, ownerId = {})", taskId, ownerId);
+    TaskImpl task = null;
+    try {
+      taskanaEngine.openConnection();
+      task = (TaskImpl) getTask(taskId);
+      TaskState state = task.getState();
+      if (state != TaskState.READY) {
+        throw new InvalidStateException(TASK_WITH_ID + taskId + IS_NOT_READY);
```

Comment:
can we please use ONE string with placeholders? 
such as string.format?
with this approach, we imply grammar and a specific language. also, this make it difficult to change the wording.  

something like 
TASK_WITH_ID_IS_NOT_READY = "Task with id %s is not in state ready";